.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ on:
 
 env:
   CACHE_VERSION: 1
+  SOLVER_CACHE_VERSION: 1
```

Comment:
We now have two different `*CACHE_VERSION` environment variables—it would be helpful to document what the difference between the two is. (Feel free to steal some existing documentation for `CACHE_VERSION` [here](https://github.com/GaloisInc/saw-script/pull/1908) in `cryptol`.)